docs/configuration.md

```diff
@@ -106,6 +104,18 @@ You can also use the `KO_DEFAULTPLATFORMS` environment variable to set the defau
 KO_DEFAULTPLATFORMS=linux/arm64,linux/amd64
 ```
 
+### Environment Variables (advanced)
+
+For ease of use, backward compatibility and advanced use cases, `ko` supports the following environment variables to
+influence the build process.
+
+| Variable         | Default Value                              | Description                                                                                                                                                                                                                      |
+|------------------|--------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `KO_DOCKER_REPO` | (not set)                                  | Container repository where to push images built with `ko` (required)                                                                                                                                                             |
+| `KO_DATA_PATH`   | `/var/run/ko` (`C:\var\run\ko` on Windows) | By convention, any contents of a directory named `<importpath>/kodata/` will be bundled into the image, and the path where it's available in the image will be identified by the environment variable `KO_DATA_PATH` (optional). |
```

Comment:
```suggestion
```

Let's remove this, since it's a bit different from the others, in that it's only involved at runtime.